src/collective/cover/tests/test_contentchooser.py

```diff
@@ -1,11 +1,12 @@
 # -*- coding: utf-8 -*-
-
 from collective.cover.testing import INTEGRATION_TESTING
 from collective.cover.config import PLONE_VERSION
 from plone import api
+from zope.component import queryUtility
+from zope.schema.interfaces import IVocabularyFactory
 
 import json
-import re
+import lxml
```

Comment:
@cewing this makes a lot of sense but we need to split to be able to test under Plone 4 and Plone 5